tensorflow/python/ops/math_ops.py

```diff
@@ -568,8 +572,8 @@ def angle(input, name=None):
   For example:
 
   ```
-  # tensor 'input' is [-2.25 + 4.75j, 3.25 + 5.75j]
-  tf.angle(input) ==> [2.0132, 1.056]
+  # tensor 'input' is [-2.25 + 4.75j, 3.25 + 5.75j], in `complex64` format
```

Comment:
To make it one step closer to runnable code you could say:

```suggestion
input = tf.constant([-2.25 + 4.75j, 3.25 + 5.75j])
```

Reply:
Good idea, change applied.